Source/ThirdParty/ANGLE/src/common/log_utils.h

```diff
@@ -248,6 +249,23 @@ std::ostream &FmtHex(std::ostream &os, T value)
 #    define EVENT(message, ...) (void(0))
 #endif
 
+// Allow us to disable assertions at runtime by setting the ANGLE_ASSERTS_OFF environment variable
+#if defined(ANGLE_ENABLE_ASSERTS)
+static inline bool IsAssertionEnabled() {
```

Comment:
This static variable access should not be inlined into all ASSERT sites. There's no downside for this to be a normal function in the .cpp file